New post: SONAR project: the end of an adventure (#5)
rero/sonar.ch: content/post/sonar-project-the-end-of-an-adventure.md

---
title: "SONAR project: the end of an adventure"
date: 2021-02-18T12:00:00+02:00
draft: false
tags: ["project"]
---

> After two years of intense work, cross-institutional exchanges, lively debates, feasibility analysis and exploratory tests, the SONAR project ended at the end of 2020. It was carried out as part of the P-5 program "Scientific Information: Access, Processing and Storage" of swissuniversities. The objective was to set up a national platform for open access scientific publications.

Reviewer: ```suggestion
After two years of intensive work, cross-institutional exchanges, lively debates, feasibility analysis and exploratory tests, the SONAR project ended at the end of 2020. It was carried out as part of the P-5 program "Scientific information: access, processing and safeguarding" of swissuniversities. The objective was to set up a national platform for open access scientific publications.
```

<!--more-->

### The whole SONAR team warmly thanks all partner institutions and colleagues who have contributed in one way or another to the project, or with whom it has exchanged ideas and experiences.

{{< figure src="/images/sonar_project_team.jpg" caption="SONAR project team" alt="Screenshot of the whole SONAR project team during a Zoom meeting" width="100%" >}}

SONAR brought together participants from the four corners (literaly!) of Switzerland:

* [RERO](https://www.rero.ch/)

>    * Miguel Moreira (principal investigator)

Reviewer: ```suggestion
   * Miguel Moreira (project manager)
```

   * Sébastien Délèze
   * Johnny Mariéthoz
   * Igor Milhit
   * Nicolas Prongué

* [HES-SO](https://www.hes-so.ch/) / [HEG&nbsp;Genève](https://www.hesge.ch/heg/)

>    * Patrick Ruch (co-investigator)

Reviewer: ```suggestion
   * Patrick Ruch (deputy project manager)
```

   * Julien Gobeill
   * Jeevanthi Liyanapathirana
   * Anouk Santos

* [FH Graubünden](https://www.fhgr.ch/)

   * Gerhard Bissels
   * Karsten Schuldt

* [USI](https://usi.ch/)

   * Silvio Bindella
   * Davide Dosi
   * Alessio Tutino

### After the project

Various services have been designed and will take over the project. The infrastructure is ready to welcome client institutions and also to progressively replace RERO DOC in the course of 2021. The transition to production will be done gradually according to the planning of the interested institutions. Various discussions are under way.

> The SONAR go-live will be announced in the next weeks.

Reviewer: ```suggestion
The SONAR go-live will be announced in the coming weeks.
```
